### PR TITLE
AncientSettings: Updates Russian translations

### PR DESCRIPTION
**Pull request by CakesTwix**

### CODE REVIEW

File: res/values-ru/ancient_strings.xml [hunk 1]
```diff
@@ -272,12 +275,12 @@
...
-    <string name="volume_summary">Настройка параметров для кнопок громкости</string>
+    <string name="volume_title">Volume Rockers</string>
+    <string name="volume_summary">Customize settings for volume buttons</string>
 
-    <!-- Volume rocker music control -->
-    <string name="volume_rocker_music_controls_title">Управление воспроизведением</string>
-    <string name="volume_rocker_music_controls_summary">Когда экран выключен, длительное нажатие клавиш регулировки громкости будет переключать музыкальные треки</string>
+    <!-- Volume Key -->
+    <string name="volume_title">Громкость</string>
+    <string name="volume_summary">Настройка параметров для кнопок громкости</string
```
Comment (AshutoshSundresh) [inline res/values-ru/ancient_strings.xml:279]: Duplicate string

**Review by AshutoshSundresh — CHANGES_REQUESTED** [review]
Please remove the duplicate string

File: res/values-ru/ancient_strings.xml (hunk 1 above)
Comment (CakesTwix) [reply res/values-ru/ancient_strings.xml:279]: Excuse me. Corrected.

**Comment by AshutoshSundresh** [comment]
Not well formed. Some strings were removed and closing brackets missing in some places.

**Pull request merged**